tests/installation/select_patterns_and_packages.pm

```diff
@@ -79,6 +79,9 @@ sub gotopatterns {
     else {
         send_key 'tab';
     }
+    # pressing up and down to have selection more visible
```

Comment:
huh, how does this work? And do we want to do this in *all* tests selecting patterns?

Reply:
Yes it works as you can see. Yes, we should do that in all tests to avoid any potential issues with first pattern selection.